PlayerUI/Util/NSWindow+Snapshot.swift

```diff
@@ -18,9 +17,7 @@ extension NSWindow {
         }
 
         let image = NSImage(cgImage: cgImage, size: frame.size)
-
         image.cacheMode = .never
-
```

Comment:
Personally I prefer a newline before the return statement to make it clear the scope is ending.